pypit/scripts/setup.py

```diff
@@ -11,11 +11,18 @@
 from __future__ import (print_function, absolute_import, division,
                         unicode_literals)
 
+import os
+import datetime
+import argparse
 import pdb as debugger
 
-def parser(options=None):
-    import argparse
+from pypit import pyputils
```

Comment:
The value of having the imports in main() is so that
one can get the help string from the parser *fast*, i.e.
without doing the imports.  I should prefer it that way
unless you are finding it is fast anyways.  Please check.

Reply:
Good point.  Here's a test with run_pypit

- A first test after making some changes to the code:
```
% time run_pypit -h
real	0m4.353s
user	0m1.785s
sys	0m0.924s
```

- Repeat without any changes:

```
% time run_pypit -h
real	0m2.937s
user	0m1.645s
sys	0m0.387s
```

 - Moving the pypit imports to inside main():

```
% time run_pypit -h
real	0m0.406s
user	0m0.329s
sys	0m0.062s
```

- Repeated:

```
% time run_pypit -h
real	0m0.383s
user	0m0.316s
sys	0m0.051s
```

So, yes, moving the imports inside definitely makes a difference to the execution time of the argparse help output.  I've moved the main imports back into main.